packages/react-native/Libraries/Lists/SectionList.d.ts

```diff
@@ -59,8 +59,13 @@ export type SectionListRenderItem<ItemT, SectionT = DefaultSectionT> = (
   info: SectionListRenderItemInfo<ItemT, SectionT>,
 ) => React.ReactElement | null;
 
+export type VirtualizedListWithoutPreConfiguredProps<ItemT> = Omit<
+  VirtualizedListWithoutRenderItemProps<ItemT>,
+  'stickyHeaderIndices'
+>;
```

Comment:
Could we rename this to something else, or avoid exporting it?

I.e. `import {VirtualizedListWithoutPreConfiguredProps} from 'react-native'` brings in a name that's a bit meaningless (really, just that SectionList doesn't want to expose it because it plugs in its own values).

Reply:
Good point! ✅
I've dropped export as it seems to be unnecessary from end user perspective.
Thanks for pointing that! 🎉🥳